backend/app/routes/auth_routes.py

```
@@ -0,0 +1,98 @@
+    
+    return jsonify({
+        'token': token,
+        'user': {
+            'id': user.id,
+            'username': user.username,
+            'email': user.email
+        }
+    })
+
+@bp.route('/user', methods=['GET'])
+def get_user():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_user`

(ANN201)